.gitignore

```diff
@@ -6,14 +6,10 @@
 /build
 /dist
 /MANIFEST
-/setuptools-3.1-py2.7.egg
-/setuptools-3.1.zip
 /stone.egg-info
 .arcconfig
 .DS_Store
 .idea
-parser.out
```

Comment:
Ya, a bit hazy, but a stone developer might create the file just by opening a python console and doing a relative import of the stone parser.